dashboard/public/css/main.css

```diff
@@ -105,3 +105,53 @@
         background-color: inherit;
     }
 }
+
+meter {
+    margin: -4em auto 1em;
+    width: 100%;
+    background: none;
+
+}
+
+meter::-webkit-meter-bar {
+    background: none;
+}
+
+
+meter[value="1"]::-webkit-meter-optimum-value {
+    background: rgb(255, 0, 0);
+}
+
+meter[value="2"]::-webkit-meter-optimum-value {
+    background: yellow;
+}
+
+meter[value="3"]::-webkit-meter-optimum-value {
+    background: orange;
+}
+
+meter[value="4"]::-webkit-meter-optimum-value {
+    background: green;
+}
+
+meter[value="1"]::-moz-meter-bar {
+    background: #ca1a64;;
+}
+
+meter[value="2"]::-moz-meter-bar {
+    background: yellow;
+}
+
+meter[value="3"]::-moz-meter-bar {
+    background: orange;
+}
+
+meter[value="4"]::-moz-meter-bar {
+    background: green;
+}
```

Comment:
Code redundancy: It can be written in 4 classes instead of 8.

Reply:
`::moz-meter-bar` selector was used as `webkit` pseudo selector isn't compatible with firefox.

https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-meter-optimum-value

Reply:
You can use comma (,) like:
meter[value="1"]::-webkit-meter-optimum-value, meter[value="1"]::-moz-meter-bar {
    background: rgb(255, 0, 0);
}